modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java

```diff
@@ -337,7 +338,7 @@ public void expectExceptionWhenOneArgsDecrementCalled_noValueFactory() {
      *                                                                         *
      **************************************************************************/
 
-    @Ignore("Need KeyboardEventFirer")
+    @Ignore("Need KeyboardEventFirer. Listener not invoked on changing value using setText method")
```

Comment:
Please file a bug for this and change the String to that bug ID, e.g., `@Ignore("JDK-nnnnnnn")`

Reply:
Created [JDK-8328701](https://bugs.openjdk.org/browse/JDK-8328701) and updated `@Ignore` comment